egret/model_library/unit_commitment/params.py

```diff
@@ -193,7 +193,7 @@ def load_params(model, model_data):
     model.LinesTo = Set(model.Buses, initialize=inlet_branches_by_bus)
     model.LinesFrom = Set(model.Buses, initialize=outlet_branches_by_bus)
 
```

Comment:
It would be useful to document the motivation for allowing negative impedences. And the end effect, e.g., inverting line flow.

Reply:
Done!